tests/foreman/ui/test_contenthost.py

```diff
@@ -324,3 +326,40 @@ def test_positive_provisioning_host_link(self):
             host_url = urljoin(settings.server.get_url(),
                                'hosts/{0}'.format(self.client.hostname))
             self.assertEqual(self.browser.current_url, host_url)
+
+    @tier3
+    def test_positive_attached_subscription_link(self):
+        """Check that the attached subscriptions link on subscriptions tab
+        points to the subscription details page
+
+        @id: bea394a9-b7c9-4128-8803-af5fe698152b
+
+        @expectedresults: Attached subscriptions link points to specific
+        subscription details page
+
+        @BZ: 1391200
+
+        @CaseLevel: System
+        """
+        with Session(self.browser):
+            self.contenthost.search_and_click(self.client.hostname)
+            self.contenthost.click(
+                tab_locators['contenthost.tab_subscriptions'])
+            self.contenthost.click(
+                tab_locators['contenthost.tab_subscriptions_subscriptions'])
```

Comment:
Yeah =) And it's technically correct, it's Subscriptions -> Subscriptions tab, the question is why do we have nested subscriptions tab :) From automation perspective it's correct as describes tab structure so i haven't touched it (although i'd rather use dot instead of underscore but that's a nitpick).